testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/AssertEvents.java

```diff
@@ -175,7 +175,7 @@ public ExpectedEvent expectRefresh(String refreshTokenId, String sessionId) {
     }
 
     public ExpectedEvent expectLogout(String sessionId) {
-        return expect(EventType.LOGOUT).client((String) null)
```

Comment:
Now there's many tests that fail (like on `ResetPasswordTest`) because we return an actual client.